### ISSUE TITLE
Radar sensor: Hi-Link LD2450

### ISSUE THREAD

**Issue by J0han3s** [ESPresense/ESPresense#1274]
### Is your feature request related to a problem? Please describe.

Integrate Hi-Link Ld2450 radar. Utilises a UART serial connection and offer more features than ld2410. Specifically it reports number of people (upto 3) and also reports presence in different zones.

### Describe the solution you'd like

Integrate the sensor, similar to how the ld2410 is, but would need to be a serial connection ( I think)/ not just binary

**Comment by baudneo** [comment]
This would be really great to have access to, Right now I have a separate ESP32-C3 for my LD2450. If ESPresense could handle it, that would be great!

**Comment by gkchim** [comment]
Good to know if this might be possible please?

**Comment by baudneo** [comment]
> Good to know if this might be possible please?

I dont think so. The current radar setup uses 1 pin as a 'motion / no motion' pin. The LD2410 has a pin out for this, the LD2450 does not have a pin out to signal 'motion / no motion'. 

They would need to add support via UART and I do not see that happening anytime soon.

**Comment by jploonstra** [comment]
> > Good to know if this might be possible please?
> 
> I dont think so. The current radar setup uses 1 pin as a 'motion / no motion' pin. The LD2410 has a pin out for this, the LD2450 does not have a pin out to signal 'motion / no motion'.
> 
> They would need to add support via UART and I do not see that happening anytime soon.

It may not be feasible on the short term, but it would be a great addition to the ESPresense platform!
This is another example: https://www.dfrobot.com/product-2795.html

**Comment by SkipperDo** [comment]
I would like to see this feature too...
The LD2410 has an UART connection too & it will be much more comfortable to add this LD2410 / LD2450 via the ESPresence platform Who is already installed in every room at my house instead installing a secondary set of ESPs in every room...
Some ESP boards (e.G. WROOM32) already has accessible a secondary UART pin set... So for small but long rooms a second LD24xx could be nice to enlarge the detection field with a single ESP... 🎉
